decisions/flow_versions.md

```diff
@@ -0,0 +1,11 @@
+# Flow versions
+
+## Decision
+The version of `flow-bin` that we use in Backpack React Native should align to the same version that React Native is using.
+
+
+## Thinking
+The reason for this is newer versions of Flow use different concepts and features that when we make changes within Backpack could actually mean we misuse and violate types that React Native expects.
+
+## Anything else
+**Note:** Any pull requests that Dependabot or Snyk raise to upgrade flow should only be done if the version being updated aligns to the version in React Native. If the version bump is higher the PR should be closed.
```

Comment:
Maybe we should exclude Flow from Dependabot, what do you reckon? That shouldn't block this PR, I'm just wondering

Reply:
I did think about this as an option and it made me wonder that if we do that are we likely to forget to ever update flow and then realise we are versions behind?

In my head I thought it might work as a good reminder when the PRs come that we may need to upgrade but I guess it would also create noise by just constantly making PRs we would close.

Both keeping and excluding it are both valid options just not sure what would be best from our maintenance PoV 🤔

Reply:
I guess we could tie it to RN upgrades, so when we upgrade RN we also upgrade Flow? And then it should be trivial to have Danger remind us to do Flow at the same time as RN upgrades

Reply:
Thats a good idea actually as newer versions of RN should hopefully be upgrading to the latest or a newer version of flow too. Will add a Dependabot exclude to this PR to keep that from being updated!

Reply:
Noice! We can do the Danger thing separately